src/content/Docs/other-resources/experimental/provider-build-via-ansible/index.md

```diff
@@ -0,0 +1,130 @@
+
+Before creating the cluster, make the following important configurations:
+
+### Configure Ephemeral Storage
+Ensure your provider is configured to offer more ephemeral storage compared to the root volume by modifying group_vars/k8s_cluster/k8s-cluster.yml on the Kubespray host.
+```yml
+containerd_storage_dir: "/data/containerd"
+kubelet_custom_flags:
+  "--root-dir=/data/kubelet"
+```
+
+### Configure Scheduler Profiles
+Add the following configuration to group_vars/k8s_cluster/k8s-cluster.yml:
+```yml
+kube_scheduler_profiles:
+- pluginConfig:
+    - name: NodeResourcesFit
+    args:
+        scoringStrategy:
+        type: MostAllocated
+        resources:
+            - name: nvidia.com/gpu
+            weight: 10
+            - name: memory
+            weight: 1
+            - name: cpu
+            weight: 1
+            - name: ephemeral-storage
+            weight: 1
+```
+
+### Enable Helm Installation
+Add the following configuration to group_vars/k8s_cluster/addson.yml:
+
+```yml
+# Helm deployment
+helm_enabled: true
+```
+
+
+## STEP 2 - Clone the Repository
+We recommend setting up the provider build steps on Ubuntu 22.04.
+
+```bash
+cd ~
+
+git https://github.com/akash-network/provider-playbooks.git
+
+cd provider-playbooks
+```
+
+## STEP 3 - Building Provider using Ansible
+Run the complete Ansible playbook with the necessary variables:
+
+```bash
+ansible-playbook playbooks.yml -i inventory.yml -e 'host=<host(s)>' -v
+```
+
+> Note: Each role requires specific variables. Refer to the Configuration Variables in the README under each role.
+
+## Example Configuration
+### Example Inventory File
+
+Create an inventory file (inventory.yml) for an All-In-One node (single control plane + etcd node):
+
+```yml
+#inventory.yml
+all:
+  hosts:
+    node1.t100.abc.xy.akash.pub:
+```
+
+### Example command
+```bash
+ansible-playbook playbooks.yml -i inventory.yml -e 'host=node1.t100.abc.xy.akash.pub tailscale_authkey='tskey-authxxxx' tailscale_hostname=node1.t100.abc.xy.akash.pub provider_name=t100.abc.xy.akash.pub provider_version=0.6.9' -v
+```
+> Note: For an All-in-One node setup, using extra vars makes sense as it's a simpler deployment with all components on a single node.
+However, when dealing with a Kubernetes cluster involving multiple nodes, using host_vars is the better approach. This allows you to define specific variables for each host in the cluster, providing more granular configuration control and better organization of your infrastructure as code.
+
+### Role-Specific Variables
+Each role in the playbook has specific configuration variables that can be set to customize your deployment. These variables can be defined in your inventory files, host_vars files, or passed directly using the -e parameter.
+
+#### Tailscale Role
+
+`tailscale_authkey`: Your Tailscale authentication key
```

Comment:
Can you please add some explanation on what's `tailscale_authkey` , what is Tailscale for, why is it used / etc.

Far not everyone is using Tailscale and it should definitely be optional, not a requirement.